internal/dao/sqlx/source_dao.go

```diff
@@ -0,0 +1,95 @@
+package sqlx
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	"github.com/RHEnVision/provisioning-backend/internal/db"
+	m "github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/jmoiron/sqlx"
+)
+
+const (
+	create     = `INSERT INTO sources (source_id, name, org_id, auth_id) VALUES ($1, $2, $3, $4) RETURNING created_at`
+	getById    = `SELECT * FROM sources WHERE source_id = $1;`
+	list       = `SELECT * FROM sources ORDER BY id LIMIT 100 OFFSET $1`
```

Comment:
Yeah LIMIT is hardcoded for now, we can fix that later and make this a URL parameter or something, not sure how this is done in OpenAPI so let's just hardcode 100 for now.